packages/shared/src/components/post/SquadPostWidgets.tsx

```diff
@@ -23,14 +23,18 @@ interface PostWidgetsProps
 const SquadCard = ({ squadSource }: { squadSource: Squad }) => {
   const { isFetched } = useAuthContext();
   const { id: squadId, handle } = squadSource;
-  const { squad = squadSource } = useSquad({ handle });
+  const { squad } = useSquad({ handle });
```

Comment:
Oh, I just saw this. Why did we end up doing another squad request in this widget?

Reply:
It's just to get the data but if you open post modal from squad page no request will be made since it will all ready be fetched. 

Addition is for the single post page where source request will have to be made but I think that is ok since we need that data. And then again if you go to squad page from that page there will be no request since it will be cached from post page. Does that make sense?